### ISSUE TITLE
vim-http

### ISSUE THREAD

**Issue by Konfekt** [sojohnnysaid/vim-restman#1]
Thank you, this is useful. Comparing to https://github.com/nicwest/vim-http, it seems that Restman supports more than one HTTP request by jumping in-between the `--` markers and restricts to Curl output (no meta data)?

**Comment by sojohnnysaid** [comment]
Thanks for checking this out @Konfekt  🤓 

Yes I am able to use all the API verbs, GET, POST, PATCH etc. etc.
Just move the cursor within the -- markers to run that specific call.

If you take a peek at the autoload folder in the repo you will see i've commented out all the echom statements where I was logging out everything. The original output had more sections like request headers, response headers etc.

Would a flag you could add to your vim config work? I can add that to my todo list.

**Comment by Konfekt** [comment]
Yes, thank you, maybe buffer-local/global by the `get(b:, 'opt', get(g:, 'opt', 'default-value))` idiom

**Comment by sojohnnysaid** [comment]
@Konfekt sorry for the delay this issue has been resolved and you'll find the changes made in the latest commit with updates in the README as well 🎉
